modules/backup-and-restore/pages/restore-cross-cluster.adoc

```diff
@@ -7,6 +7,33 @@ This page walks you through the process of restoring a cluster from a backup tha
 Restoring is an off-line operation.
 Your cluster is offline during the restore process.
 
+== gadmin backup restore -h
+
```

Comment:
This whole section should be in a code block.
See https://docs.tigergraph.com/tigergraph-server/3.11/system-management/management-commands for an example.